MVPDatabase.sql

```diff
@@ -0,0 +1,30 @@
+CREATE TABLE IF NOT EXISTS mvp_table(
+    hunt_id INT PRIMARY KEY,
```

Comment:
Needs to be AUTO_INCREMENT
```suggestion
    hunt_id INT PRIMARY KEY AUTO_INCREMENT,
```